src/App.js

```diff
@@ -1,51 +1,32 @@
-import React from 'react'
-import {
-  BrowserRouter, Route, Switch
-} from 'react-router-dom'
+import React from "react";
+import { BrowserRouter, Route, Switch } from "react-router-dom";
 //Context
-import {CtxProvider} from './Context/ctx'
-//Cookie consent
-import CookieConsent from 'react-cookie-consent'
+import { CtxProvider } from "./Context/ctx";
 //Util
-import {AppRoute} from './Helpers/helpers.js'
+import { AppRoute } from "./Helpers/helpers.js";
 //Layouts
-import DefaultLayout from './Layouts/DefaultLayout/defaultLayout'
-import PagesLayout from './Layouts/PagesLayout/PagesLayout'
-//Components
-import NotFound from './Components/NotFound/notFound'
-import About from './Components/About/about'
-import Header from './Components/Header/header'
+import PagesLayout from "./Layouts/PagesLayout/PagesLayout";
+//Pages
+import NotFound from "./pages/NotFound/notFound";
+import About from "./pages/About/about";
+import Home from "./pages/Home/home";
 
 export default function App() {
-  return (
-    <CtxProvider>
-      <BrowserRouter>
-        <Switch>
+	return (
+		<CtxProvider>
+			<BrowserRouter>
+				<Switch>
+					<AppRoute
+						exact
+						path="/about"
+						layout={PagesLayout}
+						component={About}
+					/>
 
-          <AppRoute exact path='/about' layout={PagesLayout} component={About} />
-
-          <Route path="/" exact>
-            <DefaultLayout>
-              <Header />
-
-              <CookieConsent
-                location="bottom"
-                buttonText="Sure man!!"
-                cookieName="myAwesomeCookieName2"
-                style={{ background: "#2B373B" }}
-                buttonStyle={{ color: "#4e503b", fontSize: "13px" }}
-                expires={150}
-              >
-                This website uses cookies to enhance the user experience.{" "}
-                <span style={{ fontSize: "10px" }}>This bit of text is smaller :O</span>
-              </CookieConsent>
```

Comment:
Seems you have not included the cookie consent component on the modified version. Did you intend to do so?